tests/analysis/losses/resilience_curves/test_resilience_curves.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+
+from ra2ce.analysis.losses.resilience_curves.resilience_curves import ResilienceCurves
+from ra2ce.network.network_config_data.enums.road_type_enum import RoadTypeEnum
+
+
+class TestResilienceCurve:
+    @pytest.fixture(name="valid_resilience_curves")
+    def _get_valid_resilience_curves(
+        self,
+        resilience_curves_data: list[
+            tuple[RoadTypeEnum, tuple[float, float], list[float], list[float]]
+        ],
+    ):
+        _resilience_curves = ResilienceCurves()
+        for _data in resilience_curves_data:
+            _resilience_curves.link_type.append(_data[0])
+            _resilience_curves.hazard_range.append(_data[1])
+            _resilience_curves.duration_steps.append(_data[2])
+            _resilience_curves.functionality_loss_ratio.append(_data[3])
+        return _resilience_curves
```

Comment:
Would it not make more sense (and become more readable) to use `zip`?
```suggestion
        _data_tuples = tuple(zip(*_data))
        return ResilienceCurves(link_type=_data_tuples[0], hazard_range=_data_tuples[1], ...)
```

Reply:
Nice suggestion. Applied with `_data_tuples = tuple(zip(*resilience_curves_data))` instead.